site/docs/patterns/list-filtering.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: List filtering
+layout: DetailPattern
+aliases:
+  - /salt/patterns/list-filtering
+data:
+  components: ["Input", "List", "Spinner"]
+  relatedpatterns: ["List builder"]
+---
+
+When searching through a list, a filter enables users to specify criteria and only show relevant options. This pattern introduces the filter behavior of Salt components where a list is used.
+
+  <Diagram
```

Comment:
Only list here doesn't have border compared to the rest of the images on page, should we swap?

Reply:
List doesn't have a border by default, so let's leave this image as is.